packages/experimental/RadioGroup/src/Radio/RadioTokens.ts

```diff
@@ -7,7 +7,7 @@ import type { RadioTokens } from './Radio.types';
 export const defaultRadioTokens: TokenSettings<RadioTokens, Theme> = (t: Theme) =>
   ({
     rootHorizontalPadding: globalTokens.size40,
-    borderColor: t.colors.transparentStroke,
+    borderColor: t.colors.background,
```

Comment:
This isn't a Fluent Design Token, can you use one of those instead? We should be fully using Fluent Tokens for V1 components.

Reply:
Same comment for the win32 file

Reply:
The transparentStroke token is fixed to map to the correct color. Reverted back to using this design token.